testing/helm/tests/furthercoverage.yaml

```diff
@@ -1,71 +1,17 @@
-# Default values for hpcc.
+# This values file is here to exrcise all of the different paths in the helm chart.
+# It is a copy of values.yaml, all changes are marked with #CHANGE.
+#
+# - roxie: Enabling server replicas in roxie
+# - thor: eclAgentUseChildProcesses=false
 
 global:
```

Comment:
could this and other unchanged copies from values.yaml be omitted ? since values.yaml is base and this is applied on top?

Reply:
yes.  I'll clean up some more